src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor.CoreComponents/Microsoft.CodeAnalysis.Remote.Razor.CoreComponents.csproj

```diff
@@ -25,6 +25,7 @@
       <_PublishedFiles Include="$(PublishDir)**\Microsoft.CodeAnalysis.Razor.*" />
       <_PublishedFiles Include="$(PublishDir)**\Microsoft.CodeAnalysis.Remote.Razor.*" />
       <_PublishedFiles Include="$(PublishDir)**\Microsoft.AspNetCore.*" />
+      <_PublishedFiles Include="$(PublishDir)**\Microsoft.Extensions.Logging.Abstractions.dll" />
```

Comment:
This is required because Roslyn doesn't already have `Logging.Abstractions` in their OOP process